bumble/device.py

```diff
@@ -1704,16 +1718,18 @@ async def new_connection():
             asyncio.create_task(new_connection())
 
     @host_event_handler
-    def on_connection_failure(self, connection_handle, error_code):
+    def on_connection_failure(self, transport, connection_handle, peer_address, error_code):
```

Comment:
With the modified `emit` from line 386 of `host.py`, the `connection_handle` argument is not longer present (which makes sense), so this won't work, you should remove `connection_handle` from here too.